codegen/smithy-aws-typescript-codegen/src/main/java/software/amazon/smithy/aws/typescript/codegen/AddBuiltinPlugins.java

```diff
@@ -74,10 +85,30 @@
                         .withConventions(AwsDependency.ADD_GLACIER_API_VERSION.dependency,
                                          "AddGlacierApiVersion", HAS_MIDDLEWARE)
                         .servicePredicate((m, s) -> testServiceId(s, "Glacier"))
+                        .build(),
+                RuntimeClientPlugin.builder()
+                        .withConventions(AwsDependency.SSEC.dependency, "Ssec", HAS_MIDDLEWARE)
+                        .servicePredicate((m, s) -> testServiceId(s, "S3"))
+                        .operationPredicate((m, s, o) -> testContainsMember(m, o, SSEC_OPERATIONS))
                         .build()
         );
     }
 
+    private static boolean testContainsMember(Model model, OperationShape operationShape, Set<String> expectedMemberNames) {
```

Comment:
This method can be simplified:

```java
OperationIndex operationIndex = model.getKnowledge(OperationIndex.class);
return operationIndex.getInput(operationShape)
        .filter(input -> input.getMemberNames().stream().anyMatch(expectedMemberNames::contains))
        .isPresent();
```

Reply:
fixed in 6bcea90f30616e16983326a3b87381730520931d